fork.yaml

```
@@ -78,11 +78,13 @@ def:
             - title: "Core Error definitions"
               globs:
                 - "core/error.go"
-            - title: "Gaslimit"
+            - title: "Gaslimit and EIP-1559 Params"
               description: |
                 The gaslimit is free to be set by the Engine API caller, instead of enforcing adjustments of the
-                gaslimit in increments of 1/1024 of the previous gaslimit.
-                The gaslimit is changed (and limited) through the `SystemConfig` contract.
+                gaslimit in increments of 1/1024 of the previous gaslimit. The elasticity-multiplier and
+                base-fee-max-change-denominator EIP-1559 parameters can also be set by the Engine API caller through the
+                ExtraData field.  The gaslimit and EIP-1559 parameters are changed (and limited) through the
+                `SystemConfig` contract.
               globs:
                 - "consensus/misc/eip1559/eip1559.go"
```

wondering if we also want to include the test file, it currently lands in the catch-all testing category
```suggestion
                - "consensus/misc/eip1559/*"
```